apt/meta.go

```diff
@@ -344,6 +345,13 @@ func ExtractFileInfo(p string, r io.Reader) ([]*FileInfo, Paragraph, error) {
 	case ".bz2":
 		r = bzip2.NewReader(r)
 		base = base[:len(base)-4]
+	case ".xz":
```

Comment:
please add test cases validating `.xz` support

Reply:
I added a test case for decompressing a `Packages.xz` file and checking if it contains certain files. I simply compressed `apt/testdata/af/Packages` for this.